src/ucp/rndv/rndv_rtr.c

```diff
@@ -237,12 +237,33 @@ static void ucp_proto_rndv_rtr_query(const ucp_proto_query_params_t *params,
     attr->lane_map      = UCS_BIT(rpriv->lane);
 }
 
+static void 
+ucp_proto_rndv_rtr_abort_super(void *req, ucs_status_t status, void *user_data)
+{
+    ucp_request_t *ucp_req = (ucp_request_t*)req - 1;
```

Comment:
should not we take super here?

Reply:
`ucp_proto_rndv_recv_complete` takes care of that. Maybe it would be better to change the name of the function to less confusing?

I would suggest `ucp_proto_rndv_rtr_abort_complete`. WDYT?

Reply:
a, ok